subprojects/core/src/main/java/org/gradle/groovy/scripts/internal/GradleResolveVisitor.java

```diff
@@ -80,7 +80,7 @@
  */
 public class GradleResolveVisitor extends ResolveVisitor {
     // note: BigInteger and BigDecimal are also imported by default
-    private static final String[] DEFAULT_IMPORTS = {"java.lang.", "java.io.", "java.net.", "java.util.", "groovy.lang.", "groovy.util."};
+    private static final String[] DEFAULT_IMPORTS = {"java.lang.", "java.io.", "java.net.", "java.util.", "groovy.lang.", "groovy.util.", "java.time."};
```

Comment:
I'm in favor of making this a new default import, what do you think @eskatos @bamboo?

Reply:
Sounds good to me!